### PR TITLE
Cancancan integration

### PR DESCRIPTION
**Pull request by vijayakarumudi**
Closes AgileVentures/MetPlus_tracker#90

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -52,6 +57,74 @@
...
+       @agency_admin = FactoryGirl.create(:agency_person)
+       @agency_admin.agency_roles << @aa_role
+       @agency_admin.save
+                  
+       @agency_manager = FactoryGirl.build(:agency_person)
+       @agency_manager.agency_roles << [@am_role, @aa_role]       
+       @agency_manager.save
+     end
+     it 'job seeker' do
+       expect(User.is_job_seeker?(@job_seeker.user)).to be true
```
Comment (joaopapereira) [inline spec/models/user_spec.rb]: Please add the negative tests as well
expect(User.is_job_developer?(@job_seeker.user)).to be false

**Comment by vijayakarumudi** [comment]
I will add the negative all the cases.

**Comment by vijayakarumudi** [comment]
I added all the negative cases tests for spec file.

**Comment by vijayakarumudi** [comment]
I am closing this PR

**Pull request closed without merging**